setup/steps/00_ensure_llm-d-deployer.sh

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env bash
 source ${LLMDBENCH_CONTROL_DIR}/env.sh
 
-announce "💾 Cloning and setting up llm-d-deployer..."
+if [[ $LLMDBENCH_CONTROL_WORK_DIR_SET -eq 1 && $LLMDBENCH_CONTROL_STANDUP_ALL_STEPS -eq 1 ]]; then
+  backup_suffix=$(date +%m.%d.%Y.%T)
```

Comment:
Allow ordinal sorting, and avoid special characters (`:`).
```suggestion
  backup_suffix=$(date +"%Y-%m-%d_%H.%M.%S")
```

Reply:
`date +%m.%d.%Y.%T` creates timestamps like `07.09.2025.18:02:26`, the colons are not supported by all OSs (like Windows).

`date +"%Y-%m-%d_%H.%M.%S"` uses dots instead, with timestamps like `2025-07-09_18.02.26`, and has the advantage of always sorting temporally (most significant digits first).

Reply:
Fixed